coriolis/osmorphing/oracle.py

```diff
@@ -22,6 +22,14 @@ def _run_dracut(self):
         self._run_dracut_base('kernel')
         self._run_dracut_base('kernel-uek')
 
+    def _find_oracle_repos(self, repos_to_enable):
+        for repo in repos_to_enable:
+            cmd = 'egrep "^\[.*%s.*\]$" -R /etc/yum.repos.d | cut -f2 -d:'
+            available_repos = self._exec_cmd_chroot(
+                cmd % repo).decode().splitlines()
+            self._enable_repos += [
```

Comment:
Having "magical" class attributes only set during calls to certain methods is bad IMO.

I know this was here before, but can you please refactor the `_enable_oracle_repos` to be called in `install_packages` (both here, in the base RHEL tools, as well as checking all the provider OSMorphing tools for this.